juju/juju
# Model allocations

## Description of change
This changes the behavior of juju commands that manage budgets and allocations. 
The ```juju update-allocation``` command is removed. The ```juju allocate``` command now allows to change allocations associated with models, not applications. The ```juju show-budget``` command also no longer provides per-application break-down of usage.

This change is needed to implement the changed approach to managing budgets in juju models.

## QA steps

After creating a juju model, run ```juju sla standard --budget 500```. Then run ```juju allocate 1000```. 
The output of ```juju show-budget``` should reflect that the allocation of the current model is 1000.

## Documentation changes

This change will need to be documented along with juju sla support.

## Bug reference

None.
